iepy/preprocess/ner/regexp.py

```diff
@@ -0,0 +1,107 @@
+import re
+import codecs
+
+from nltk.text import TokenSearcher as NLTKTokenSearcher
```

Comment:
This import looks unused since (apparently) no method from `NLTKTokenSearcher` is used.